store/sqlstore/post_store.go

```diff
@@ -115,18 +115,18 @@ func (s *SqlPostStore) Save(post *model.Post) (*model.Post, *model.AppError) {
 		post.Type != model.POST_ADD_TO_CHANNEL && post.Type != model.POST_REMOVE_FROM_CHANNEL &&
 		post.Type != model.POST_ADD_TO_TEAM && post.Type != model.POST_REMOVE_FROM_TEAM {
 		if _, err := s.GetMaster().Exec("UPDATE Channels SET LastPostAt = GREATEST(:LastPostAt, LastPostAt), TotalMsgCount = TotalMsgCount + 1 WHERE Id = :ChannelId", map[string]interface{}{"LastPostAt": time, "ChannelId": post.ChannelId}); err != nil {
-			mlog.Error(fmt.Sprintf("Error updating Channel LastPostAt: %v", err.Error()))
+			mlog.Error("Error updating Channel LastPostAt:", mlog.Err(err))
```

Comment:
Probably here we can replace the `:` at the end with a `.`.

Reply:
done